### DISCUSSION TITLE
Signal plot start callback example

### DISCUSSION THREAD

**Discussion by leemagnusson** [facontidavide/PlotJuggler#621] [Q&A]
I'm working on a plugin for data streaming. The plugin examples are straightforward to follow but I am stuck on one challenge. I'd like to be able put all the possible timeserieses that I have in the Timeseries list. But then only stream at high frequency ones that are added to a plot. Is there an example how I might be able to accomplish this?

**Comment by facontidavide** [comment]
Currently, that is not possible.
The plugin can not know which series are added to plots

**Reply by leemagnusson** [reply]
Cool thanks for letting me know. Is there any way from the plugin to interact with the signals in the timeseries list? Like respond to right click? For some reason I seem to want to set it up to use that list to display all my signals and then modify them rather than having a separate dialog box, though the dialog box seems straightforward.

**Reply by facontidavide** [reply]
This kind of interaction is "potentially" possible, but not currently.
If you or anybody wants to sponsor this development, let me know.

**Reply by leemagnusson** [reply]
Unfortunately I don't have any good ability to sponsor right now but I can look into it. Are you able to quote an amount you may be looking for? It's part of an unfunded open source project of my own - obot.dev. I could send you a motor controller and motor than can stream data over usb at 10 kHz if that is useful to you.

**Reply by facontidavide** [reply]
when I need to stream a lot of data, I usually use **pal_statistics**, that PlotJuggler supports.
Unless you have pointclouds and images, just stream everything with it
